redirects.json

```diff
@@ -0,0 +1,117 @@
+[
+    {
+        "fromPath": "/careers/people-management",
+        "toPath": "/careers/human-resources/",
+        "status": 301
```

Comment:
Do I need to use status here or as we have it here https://github.com/binary-com/deriv-com/pull/5533/files#diff-a6f3904c41211be170d9b0eb87e49ea8d02d0fd5a7224d6772174b425dacf90fR14 in gatsby-node.js file is sufficient ?

Reply:
Yes, not needed here

Reply:
https://github.com/binary-com/deriv-com/pull/5533/files#r1328571235